src/main/java/pl/put/poznan/transformer/transformation/ExpandAcronymsTextTransformation.java

```diff
@@ -0,0 +1,42 @@
+package pl.put.poznan.transformer.transformation;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ExpandAcronymsTextTransformation implements TextTransformation{
+    public static final String NAME = "expand";
+
+    private Map<String, String> map = new HashMap<>() {{
```

Comment:
maybe the field could be static and final using Map.of()...
anyway name should be more meaningful

Reply:
good point, name of the variable changed